rosdoc2/verbs/build/builders/doxygen_builder.py

```diff
@@ -77,6 +77,17 @@ def __init__(self, builder_name, builder_entry_dictionary, build_context):
 
         assert self.builder_type == 'doxygen'
 
+        # If the build type is not `ament_cmake/cmake`, there is no reason
+        # to create a doxygen builder.
+        if (
+            self.build_context.build_type not in ('ament_cmake', 'cmake') and
+            not self.build_context.always_run_doxygen
+        ):
+            logger.info(
+                f"The package build type is not 'ament_cmake' or 'cmake', hence the"
+                f"'{self.builder_type}' builder was not configured")
+            return None
```

Comment:
`return None` is kind of odd here since it's a constructor. Maybe just `return`, which is the same thing really, but at least it implies there's no return type that matters here.